From: Yang Li <yang.lee@linux.alibaba.com>
Subject: crash: remove duplicated include in vmcore_info.c
Date: Fri, 26 Jan 2024 08:57:44 +0800

The header files kexec.h is included twice in vmcore_info.c,
so one inclusion can be removed.

Link: https://lkml.kernel.org/r/20240126005744.16561-1-yang.lee@linux.alibaba.com
Signed-off-by: Yang Li <yang.lee@linux.alibaba.com>
Cc: Baoquan He <bhe@redhat.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 kernel/vmcore_info.c |    1 -
 1 file changed, 1 deletion(-)

--- a/kernel/vmcore_info.c~crash-split-vmcoreinfo-exporting-code-out-from-crash_corec-fix
+++ a/kernel/vmcore_info.c
@@ -13,7 +13,6 @@
 #include <linux/memory.h>
 #include <linux/cpuhotplug.h>
 #include <linux/memblock.h>
-#include <linux/kexec.h>
 #include <linux/kmemleak.h>
 
 #include <asm/page.h>
_
